src/site/content/en/angular/setup-dynamic-server-side-rendering-with-angular/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Set up dynamic server-side rendering with Angular"
+subhead: |
+  Dynamically server-side rendering your pages per route using Angular Universal schematics.
+hero: angular-universal-motorcycle.jpeg
```

Comment:
As with the other post, I'm not quite seeing how this image relates to the post. Maybe omit?